File: Assignment/Prime.js
function prime(n) {
    var i = 0;
    var C = 0;
    for (i = 1; i <= n; i++) {
        if (n % i == 0)
            (C++);
    }
    if (C == 2) {
        console.log("NOt Prime");
    }
    else {
        console.log("prime");
    }
}
var n = 12;
prime(n);

File: Assignment/prime.c
#include<stdio.h>
int main()
{
    int i;
    int n=0;
    int x = 11;
    for(i=1;i<=x;i++)
    {
        if(x%i==0)
        {
            n++;
        }
    }
        if(n==2)
        {
            printf("prime");
        }
        else
        {
            printf("Not prime");
        }

    

    return 0;
}